InvenTree/templates/js/translated/stock.js

```diff
@@ -2226,6 +2226,7 @@ function loadStockLocationTable(table, options) {
 
     if (tree_view) {
         params.cascade = true;
+        params.depth = MAX_DEPTH;
```

Comment:
I would also say that if we are in tree view, we *must* set the `cascade=true` parameter. Otherwise the tree-view part doesn't work :)